Dockerfile.debug

```
@@ -8,7 +8,7 @@ ENV GOPATH /go
 WORKDIR /go/src/headscale
 
 RUN apt-get update \
-  && apt-get install --no-install-recommends --yes less jq \
+  && apt-get install --no-install-recommends --yes less jq ca-certificates \
   && rm -rf /var/lib/apt/lists/* \
   && apt-get clean
```

```suggestion
  && rm -rf /var/lib/apt/lists/*
```
that is run automatically in debian images